Formula/sqlcmd.rb

```diff
@@ -0,0 +1,20 @@
+class Sqlcmd < Formula
+  desc "Microsoft SQL Server command-line interface"
+  homepage "https://github.com/microsoft/go-sqlcmd"
+  url "https://github.com/microsoft/go-sqlcmd/archive/refs/tags/v0.9.1.tar.gz"
+  sha256 "35a2d165ae6e1f39033ef7ab9ef3b0b205dc1b6b5583912a082297e93ad1c8ab"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/sqlcmd"
```

Comment:
```suggestion
    ENV["CGO_ENABLED"] = "0"
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/sqlcmd"
```
Not a fan of white space here.